gemmbench/gemm_bench.py

```diff
@@ -76,10 +76,15 @@ def compile_gemm(tag, config, kernel_dir, vmfb_dir, target, extra_compiler_args,
         default=None,
         help="Directory to which executable files will be dumped."
     )
+    parser.add_argument(
+        "--raw_accumulators",
+        action='store_true',
```

Comment:
I think we still need the default to make sure this remains a bool when not set (instead of `None`)
```suggestion
        action='store_true',
        default = False,
```

Reply:
That is actually implied by `store_true`:

https://docs.python.org/3/library/argparse.html
> 'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively: